### ISSUE TITLE
Published content cannot be saved as draft

### ISSUE THREAD

**Issue by clickwork-git** [bludit/bludit#892]
With Bludit 3.5 published content cannot be "reseted" and be saved as draft.

**Comment by dignajar** [comment]
This is not a bug, is a missing feature, because I never considerate a published page can be again a draft page, what I'm thinking here, if you publish a book you can not move to draft, do you get my point ?

**Comment by clickwork-git** [comment]
Rearranging projects I sometimes change between published and draft... Also I use published pages to print and proof read, and set them back to draft.

**Comment by G3N-es** [comment]
I save as published for error every day.... Only because is de first option and is easy make a mistake.
Also, the color gray disincentives to be pulsed, while the blue incentive.
Why? Because the meaning of the gray color in user interfaces is button not active, especially if the other buttons have another color.

Unconsciously, you press publish, and then there is no possibility of going back, except by editing the database by hand.

There are infinite reasons to want to convert an article into a published or draft ...

I want you to read it only in the specific moments that I decide.
For example, it is a course that I will give only in January and May (the rest of the year is in draft).

A contest in which the visitors that come must read it at a certain moment ... which allows me a direct interaction with the public from social networks ...

In short the solution:
Any type of content should have a boleana option (drop-down, handle, checkbox ...) between published or unpublished.

Draft is not a content type, but is a state (because in fact draft is not a type of content, the type of content is the same.) The article is in a different state ...

A state in which you can find any type of content, page, product, survey ...

Any type of content should have a boleana option (drop-down, handle, checkbox ...) between published or unpublished.

**Comment by dignajar** [comment]
When the page is already published, the button are these, or we can change the button `Save as draft` for `Converto to draft` or similar.

<img width="1284" alt="screenshot 2018-12-31 at 17 13 14" src="https://user-images.githubusercontent.com/507352/50563701-70418480-0d1f-11e9-8311-1fd0a91754dc.png">

**Comment by G3N-es** [comment]
The second option is more logical.

In the immediate future we will need to create new types of content.

In my particular case, I want to create a virtual store plugin.

For example, the type of content "product", will require the draft option.

Sure sooner or later other developers will create new types of content, and certainly need the draft status. As it is now, they should stop being the content they are, to become content draft. They would lose their identity as a type of content and therefore their special fields.

For these reasons, my proposal is that "draft", "scheduledy" and "stycky" disappear as content types (because they are not) and become generic properties of the base page content, since it is interesting that these properties are inherited by any type of content that is created later.

I think it's the best way to expand and make the project more usable.

<img id="jscoverImagePreview" class="mx-auto d-block w-100" alt="Cover image preview" src="http://g3n.es/bl-content/uploads/pages/a46bbd3e1390e3c5416c3f3057379508/propuesta.png">

When I want, the first one appears in the list of articles, and when I want the draft to disappear, waiting to be used again.

Thanks for taking it into consideration.

**Comment by BlakesHeaven** [comment]
Good discussion and I support most of what @G3N-es says. I think that the use of  type and status are being merged together causing confusion.
This is how I see it:
1. STATUS:- Is it published or not, if not the page is draft - have drop-down with choice of one or the other. Only really need a 'Save' button. STATUS is just one part of whether a page is seen by the public.

2. TYPE:- Normal, Sticky, Static.  Just that and nothing to do with STATUS. 

3. SCHEDULED:- not a status, not a type, just a state describing a page with STATUS of "Published" and a published date in the future; I.e., works in conjunction with, but NOT tied to STATUS.

Also linked; as per issue #700, neither a STATUS of "draft", nor a future scheduled page date, should restrict whether a page can be used as a Parent or not. All pages of the same TYPE, not being a child, should be allowed to be a Parent.

**Comment by dignajar** [comment]
Hi,
I just uploaded a solution to GitHub for the main issue.

![publish-to-draft](https://user-images.githubusercontent.com/507352/50690330-231a2700-102d-11e9-80b7-753b289ce9f7.gif)

I know you have some ideas to improve the status / type but for that we need to change the database and the code for different classes (ex. page.class.php and other that depends on it), so please create an issue in Github with the request, then we can discuss it in that issue and try to implement for some major release for example for Bludit v4.

**Comment by G3N-es** [comment]
Thanks. It is a good solution: Publish<>Draft.
I will create the issue.

Feliz año :-)

**Comment by dignajar** [comment]
Implemented in Bludit v3.6. Thanks.
